packages/core/src/GondelDomUtils.ts

```diff
@@ -50,69 +93,104 @@ export function stopComponents(domContext?: ArrayLikeHtmlElement, namespace: str
   const components = findComponents(domContext, undefined, namespace);
   const rootComponent = domContext && getComponentByDomNode(domContext);
   if (rootComponent) {
-    components.unshift(rootComponent);
+    components.unshift(rootComponent as any);
   }
   components.forEach(component => component.stop!());
 }
 
+export function isComponentMounted(
+  domNode: ArrayLikeHtmlElement,
+  namespace: string = "g"
+): boolean {
+  const firstNode = getFirstDomNode(domNode);
+  const gondelComponent = (firstNode as any)[getGondelAttribute(namespace)];
+
+  return gondelComponent && gondelComponent._ctx;
```

Comment:
this is not of type `boolean` but of type `dom element` so `isComponentMounted(x) === true` would always be false